oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/otlp/OpenTelemetryMetricRequestProcessor.java

```diff
@@ -187,7 +187,7 @@ private static Map<Double, Long> buildBucketsFromExponentialHistogram(
         final Map<Double, Long> result = new HashMap<>();
         double base = Math.pow(2.0, Math.pow(2.0, -scale));
         if (base == Double.POSITIVE_INFINITY) {
-            if (log.isDebugEnabled()) {
+            if (log.isWarnEnabled()) {
                 log.warn("Receive and reject out-of-range ExponentialHistogram data");
             }
```

Comment:
You don't have extra operation besides the logout, then usually, with or w/o `isWarnEnabled`, they are nearly same.

Reply:
> You don't have extra operation besides the logout, then usually, with or w/o `isWarnEnabled`, they are nearly same.

So should I delete the if judge statement? And for the error, I think it is because I use jdk since 16. I will try to use jdk 11.

Reply:
JDK 11 is our minimum JDK requirement. You could set IDE(IntelliJ Language level) to JDK 11 for code APIs.
But JDK16 is not an LTS version, JDK 11 and 17 are a better choice.

Reply:
Unless you built more complex logic in the logging, you don't need this for every debug.